Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-name = "ledger_manager"
+name = "baaca"
```

Comment:
Can we not mix renaming this project with the introduction of the GUI? I like the name but if we're going to use Nicolas' last name i'd like to 1) let him know first 2) remove the strong wording i have been using.

Reply:
Ok i see you've been using it in the user interface and all. Don't rename everything but still i'd rather you don't change the name in the Cargo.toml yet.